fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -205,6 +196,9 @@ class MirabufSceneObject extends SceneObject {
         return mesh
     }
 
+    /**
+     * Changes the mode of the mirabuf object from being interacted with to being placed.
+     */
     public EnableTransformControls(): void {
```

Comment:
What happens if this is called when the transform gizmo is already enabled? Will it handle it accordingly?